components/calendar/src/coptic.rs

```diff
@@ -257,3 +258,19 @@ impl DateTime<Coptic> {
         })
     }
 }
+
+fn year_as_coptic(year: i32) -> types::FormattableYear {
+    if year > 0 {
+        types::FormattableYear {
+            era: types::Era(tinystr!(16, "ad")),
```

Comment:
Question: why are you using "ad" for coptic?

Reply:
probably was a copy paste error when the calendar was implemented. Probably should be  "am" or something

Reply:
This would be a topic for https://github.com/FrankYFTang/proposal-intl-temporal

Reply:
https://github.com/FrankYFTang/proposal-intl-temporal/issues/1